mantidproject/mantid
# Engineering Diffraction GUI - add option to not overplot fit results

Previously, once a fit had been done on a run in the ED GUI, the reconstructed peaks were always overplotted over the run. A checkbox has been added to the GUI to hide the peaks, should the user wish to do so.

**To test:**

Make sure unit tests pass locally, code review

Some sample data is [here](https://github.com/mantidproject/mantid/files/1460806/ENGINX_focused_runs.zip).

Load up the ED GUI and go to the **Fitting** tab.

Load in a few runs, and do a fit on one of them. Check the **Plot fitted peaks** box and then select that run - you should see the fit results overplotted. Selecting another run without unchecking the box should give a warning, and plot the run without any reconstruct peaks. On the run you did a fit on, make sure the peaks disappear and reappear as you toggle the checkbox on and off

Fixes #21377 

**Release Notes** 
Does not need to be in the release notes, as it is a fairly minor piece of new functionality

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
